task/script.js

```diff
@@ -1,9 +1,285 @@
 const API_BASE_URL = window.API_BASE_URL;
+const SKILLS_API = 'https://services.realdevsquad.com/skilltree/v1/skills';
+
 const suggestedUsers = [];
 const allUser = [];
+
+const params = new URLSearchParams(window.location.search);
+const isDev = params.get('dev') === 'true';
+
+const featureUrlField = document
+  .getElementById('featureUrl')
+  .closest('.inputBox');
+
+if (isDev && featureUrlField) {
+  featureUrlField.style.display = 'none';
+}
+const featureRadio = document.getElementById('feature')?.closest('.inputBox');
+
+if (isDev && featureRadio) {
+  featureRadio.style.display = 'none';
+}
+const taskLevelDiv = document
+  .querySelector('.inputBox label[for="taskLevel"]')
+  .closest('.inputBox');
```

Comment:
- instead of using a nested query to select a div can we target the div directly? this will help us prevent any unnecessary changes if something inside the inputBox changes and help us get rid of the `closest` method. what do you think?

Reply:
Have done this, please do review again